include/input_handler.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <glad/glad.h>
+#include <GLFW/glfw3.h>
+#include <window.h>
+#include <functional>
+
+namespace arch{
+    class InputHandler{
+    public:
+        InputHandler(const InputHandler&) = delete;
```

Comment:
Either remove these 2 lines (since it is very unlikely someone would use them unintentionally) or add also `operator=(InputHandler&)` and `operator=(InputHandler&&)`